<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>211</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bedarfshalt - Zug</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_connected_to">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Nicht verbunden.</string>
    </property>
   </widget>
   <widget class="QLabel" name="value_station_name">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="value_flag_stop">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>311</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Kein Haltewunsch empfangen.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_ne5">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="declineButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>150</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>239</red>
          <green>41</green>
          <blue>41</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>239</red>
          <green>41</green>
          <blue>41</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>239</red>
          <green>41</green>
          <blue>41</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="text">
     <string>Haltewunsch ablehnen</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
